compiler/optimizer/OMROptimizer.cpp

```diff
@@ -2787,7 +2789,8 @@ void OMR::Optimizer::doStructureChecks()
 
 bool OMR::Optimizer::getLastRun(OMR::Optimizations opt)
    {
-   TR_ASSERT(_opts[opt], "Optimization manager for %d should be initialized first", opt);
+   if (!_opts[opt])
```

Comment:
I'm not sure why this change was needed? Does the optimizer ever ask about last run for an opt that isn't in a strategy list (and that presumably would be initialized if it were in a strategy list) ?